config/services.php

```diff
@@ -39,10 +39,50 @@
      * Socialite providers
      */
 
+    'azure' => [
+        'client_id' => env('AZURE_CLIENT_ID'),
+        'client_secret' => env('AZURE_CLIENT_SECRET'),
+        'redirect' => env('AZURE_REDIRECT_URI', '/auth/azure/callback'),
+    ],
+
+    'facebook' => [
+        'client_id' => env('FACEBOOK_CLIENT_ID'),
+        'client_secret' => env('FACEBOOK_CLIENT_SECRET'),
+        'redirect' => env('FACEBOOK_REDIRECT_URI', '/auth/facebook/callback'),
+        'logo' => '/img/auth/facebook.png',
```

Comment:
The others don't have a logo?

Reply:
They default to `img/auth/{provider}.svg`